applications/mlflow/charts/mlflow/examples/object-storage/external/existing-secret.yaml

```
@@ -0,0 +1,24 @@
+# MLFlow with external artifact store (AWS S3) using existing secret
+
+# Disable embedded MinIO since we'll use external S3
+minio:
+  enabled: false
+
+# Configure MLflow to use external S3 as artifact store
+mlflow:
+  trackingServer:
+    mode: serve-artifacts
+    artifactsDestination: "s3://your-s3-bucket/mlflow"
+    
+  artifactStore:
+    s3:
+      enabled: true
+      # Secret containing AWS credentials
+      existingSecret: "mlflow-s3-credentials"
+      ignoreTls: false
+      external:
+        enabled: true
+        protocol: https
+        host: "s3.amazonaws.com"
+        port: 443
+        ignoreTls: false 
```

needs newline at EOF
```suggestion
        ignoreTls: false
```